<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForceDisplayDialog</class>
 <widget class="QDialog" name="ForceDisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pressure Integration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Field</string>
       </property>
       <property name="buddy">
        <cstring>cbSelectField</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSelectField">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Moment Reference:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbRefX">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="prefix">
        <string>X = </string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-99999999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000000013287555072.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbRefY">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="prefix">
        <string>Y = </string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-99999999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000000013287555072.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbRefZ">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="prefix">
        <string>Z = </string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-99999999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000000013287555072.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="twDisplay">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Section</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fx</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fy</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fz</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mx</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>My</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mz</string>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbExport">
       <property name="text">
        <string>E&amp;xport...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>ForceDisplayDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
